src/client/src/models/Curriculum.js

```diff
@@ -0,0 +1,32 @@
+export default class Curriculum {
+    constructor(id, title, description, colour, materials, steps) {
+        this.id = id;
+        this.title = title;
+        this.description = description;
+        this.colour = colour;
+        this.curriculums = [];
+        this.materials = [];
+        this.steps = [];
+    }
+
+    addCurriculum(curriculum) {
+        this.curriculums.push(curriculum);
+    }
+
+    removeCurriculum(curriculum) {
+        for (let i = 0; i < this.curriculums.length; i++) {
+            if (curriculum.id == this.curriculums[i].id) {
+                this.curriculums.splice(i, 1);
+                break;
+            }
+        }
+    }
+
+    modifyCurriculum(title, description, color, materials, steps) {
```

Comment:
As the code will get bigger, I think we should be consistent from the beginning. For the Employee class, we decided to call it the "edit()" function, but here it is modifyCurriculum(). Same goes for addCurriculum() (or simply add() ) and removeCurriculum() (or simply remove() )?

@merekorlowski  any thoughts?

Reply:
Yes I agree. However in this case, we will be removing add and remove from this class, as there will not be an array of curriculums in the Curriculum object.